independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/tooling/dockerfiles/base/src/main/docker/Dockerfile.tpl.qute.jvm

```diff
@@ -1,11 +1,4 @@
-{#include Dockerfile-layout type='jvm'}
-    {#quarkusbuild}{buildtool.cli} {buildtool.cmd.package}{/quarkusbuild}
-    {#copy}
-# We make four distinct layers so if there are application changes the library layers can be re-used
-COPY --chown=185 {buildtool.build-dir}/quarkus-app/lib/ /deployments/lib/
-COPY --chown=185 {buildtool.build-dir}/quarkus-app/*.jar /deployments/
-COPY --chown=185 {buildtool.build-dir}/quarkus-app/app/ /deployments/app/
-COPY --chown=185 {buildtool.build-dir}/quarkus-app/quarkus/ /deployments/quarkus/
-    {/copy}
-{/include}
-
+{#include Dockerfile-jar
+  type='jvm'
```

Comment:
Here, I think we should use `fast-jar` and not `jvm` because that's really what differentiate the two in the context of this include: either you have a fast-jar or an uber-jar. The template file can stay as is. That's really about changing the `type` in this file and the one that is testing it.

Reply:
@vsevel did you see this comment? It looks like this one is nearly ready to get merged?